appveyor.yml

```diff
@@ -15,7 +15,7 @@ install:
   - cmd: conda info -a
   - cmd: python --version
   # pyslim is not yet on conda-forge.
-  - cmd: python -m pip install pyslim
+  - cmd: python -m pip install "pyslim>=0.401"
```

Comment:
I don't think this is needed, is it? We'll always install the latest version on appveyor, and we only need to pin when we're worried about people having older versions lying around.